starport/pkg/templateutils/templateutils.go

```diff
@@ -0,0 +1,26 @@
+package templateutils
```

Comment:
```suggestion
package names
```

The content of this pkg seems like not directly about templates but about naming things.

Reply:
I named it like this because it was originally in the `templates` package but we need this function for ts templates as well.

I don't feel like `names` is related to the package. What about `strutils` ?

Reply:
utils word is not preffered much in the package names. scoped names are always better.

https://blog.golang.org/package-names
https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common

Reply:
we also should have pkg/xstrings

Reply:
Ok then it's fine to go with name

Reply:
> we also should have pkg/xstrings

Oh yes that can be even better